### PR TITLE
Include kubectl in Dockerfile

### PR DESCRIPTION
**Pull request by jessitron**
this was necessary for docs-sdm to work, so i figure it should be here too.

### CODE REVIEW

**Comment by ddgenome** [comment]
But it is not really necessary for an empty SDM since many people who will be using this will not be using kubernetes.

**Comment by cdupuis** [comment]
It is very likely that the kubectl is needed as it is needed by the SDMs infrastructure once deployed on Kube with the provided deployment spec.

File: Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ FROM ubuntu:18.04
 LABEL maintainer="Atomist <docker@atomist.com>"
 
 RUN apt-get update && apt-get install -y \
-        curl \
+    curl \
```
Comment (ddgenome) [inline Dockerfile]: I'd prefer to leave the whitespace as is.

File: Dockerfile [hunk 2]
```diff
@@ -28,8 +28,8 @@ ENTRYPOINT ["dumb-init", "node", "--trace-warnings", "--expose_gc", "--optimize_
 CMD ["node_modules/.bin/atm-start"]
 
 RUN apt-get update && apt-get install -y \
-        build-essential \
-        git \
+    build-essential \
+    git \
```
Comment (ddgenome) [inline Dockerfile]: I'd prefer to leave the whitespace as is.

File: Dockerfile [hunk 3]
```diff
@@ -39,6 +39,10 @@ RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - \
     && apt-get install -y nodejs \
     && rm -rf /var/lib/apt/lists/*
 
+RUN curl -sL -o /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.8.12/bin/linux/amd64/kubectl \
```
Comment (ddgenome) [inline Dockerfile]: Perhaps we should use a more recent version?

**Comment by jessitron** [comment]
I added it to the docs instead.

**Pull request closed without merging**